x-pack/plugin/esql/qa/server/multi-clusters/src/javaRestTest/java/org/elasticsearch/xpack/esql/ccq/MultiClustersIT.java

```diff
@@ -135,6 +135,14 @@ private Map<String, Object> run(String query, boolean includeCCSMetadata) throws
         return resp;
     }
 
+    private Map<String, Object> runWithColumnarAndIncludeCCSMetadata(String query) throws IOException {
+        Map<String, Object> resp = runEsql(
+            new RestEsqlTestCase.RequestObjectBuilder().query(query).includeCCSMetadata(true).columnar(true).build()
+        );
+        logger.info("--> query {} response {}", query, resp);
```

Comment:
Leftover?

Reply:
No, I'm still using that method to have a test that does both columnar: true and includes_ccs_metadata:true.